starters/next13-zustand-bulma/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "next13-zustand-bulma",
+  "description": "NextJS 13, Zustand, and Bulma",
```

Comment:
Can we also add the `hasShowcase` key and set it to false here?
That is mentioned in the readme and it doesn't look like we have that in the `package.json` file